plugin/core/windows.py

```diff
@@ -410,7 +410,7 @@ def handle_show_message(self, session: Session, params: Any) -> None:
         sublime.status_message("{}: {}".format(session.config.name, extract_message(params)))
 
     def update_diagnostics_panel_async(self) -> None:
-        to_render = []  # type: List[str]
+        to_render = ""  # type: str
```

Comment:
What makes you think appending strings is faster than joining a list?

Reply:
I haven't tought about what is faster.
I saw the way the content for rename and reference was created, 
so I wanted to to use string concatenation to make it consistent for all three:

https://github.com/sublimelsp/LSP/blob/st4000-exploration/plugin/references.py#L123
https://github.com/sublimelsp/LSP/blob/st4000-exploration/plugin/rename.py#L163

That is the reason why I did that. :)

Reply:
`''.join` is the preferred way to build up chunks of a larger string: https://stackoverflow.com/a/1350289

Reply:
List or string?
Something tells me that it should be a list :)

Reply:
done with 02d6859